### ISSUE TITLE
Bug: onSelectSlot does not work on iPad

### ISSUE THREAD

**Issue by jenstroeger** [intljusticemission/react-big-calendar#1399]
Using a mobile device, navigate to your own [demo page](http://intljusticemission.github.io/react-big-calendar/examples/index.html#selectable), switch to the “Create events” example and tap around… slots can not be selected. However, existing events respond to taps.

Verified on iPad 4 with Chrome and Safari, and on Android Phone with Chrome.

This issue seems to be a duplicate of issues https://github.com/intljusticemission/react-big-calendar/issues/972 and https://github.com/intljusticemission/react-big-calendar/issues/364, both of which were closed without explanation or fix.

**Comment by dpikt** [comment]
To clarify, this is just for taps right? I think tap+drag works on mobile.

**Comment by jenstroeger** [comment]
@dpikt, correct. I don’t think I tried tap and drop after tap alone didn’t work.

**Comment by jenstroeger** [comment]
This issue is _not_ stale, and AFAIK it has not been addressed either.

**Comment by jenstroeger** [comment]
This issue continues to be there, it’s not being fixed, and should _not_ be marked stale just because it’s not being addressed.

**Comment by mPooja-15** [comment]
> Using a mobile device, navigate to your own [demo page](http://intljusticemission.github.io/react-big-calendar/examples/index.html#selectable), switch to the “Create events” example and tap around… slots can not be selected to create new events. However, existing events respond to taps.
> 
> Verified on iPad 4 with Chrome and Safari, and on Android Phone with Chrome.
> 
> This issue seems to be a duplicate of issues #972 and #364, both of which were closed without explanation or fix. Also somewhat related is issue #1005.

you can use        longPressThreshold={20} 
it will work on iPad 
Thanks
